### PR TITLE
Show bad images

### PR DESCRIPTION
**Pull request by AlexeyTrekin**
Unacceptable images are shown in source list, but produce visible error when selected

### CODE REVIEW

File: mapflow/mapflow.py [hunk 1]
```diff
@@ -1518,6 +1518,14 @@ def create_processing_request(self,
         wd_id = self.workflow_defs.get(wd_name).id
         try:
             self.check_processing_ui(allow_empty_name=allow_empty_name)
+            if imagery:
+                # raster layer selected is local tiff
+                if not helpers.raster_layer_is_allowed(imagery):
+                    raise BadProcessingInput(self.tr("Raster image is not acceptable. "
+                                                     " It must be a tiff file, have pixel size less than {size}"
```
Comment (Godnik) [inline mapflow/mapflow.py:1525]: " It must be a Tiff file, pixel size less than {size}"

File: mapflow/mapflow.py [hunk 2]
```diff
@@ -1518,6 +1518,14 @@ def create_processing_request(self,
...
+                                                     " and memory size less than {memory}"
```
Comment (Godnik) [inline mapflow/mapflow.py:1526]: " and file size less than {memory}"

**Review by Godnik — CHANGES_REQUESTED** [review]
Tested, images are visible. I'd propose slight edits in the "file error" notification text

**Comment by AlexeyTrekin** [comment]
Done

**Pull request merged**